nautobot/core/graphql/schema.py

```diff
@@ -153,6 +159,45 @@ def extend_schema_type_custom_field(schema_type, model):
     return schema_type
 
 
+def extend_schema_type_computed_field(schema_type, model):
+    """Extend schema_type object to had attribute and resolver around computed_fields.
+    Each computed field will be defined as a first level attribute.
+
+    Args:
+        schema_type (DjangoObjectType): GraphQL Object type for a given model
+        model (Model): Django model
+
+    Returns:
+        schema_type (DjangoObjectType)
+    """
+
+    cfs = ComputedField.objects.get_for_model(model)
+    prefix = ""
+    if settings.GRAPHQL_COMPUTED_FIELD_PREFIX and isinstance(settings.GRAPHQL_COMPUTED_FIELD_PREFIX, str):
+        prefix = f"{settings.GRAPHQL_COMPUTED_FIELD_PREFIX}_"
+
+    for field in cfs:
+        field_name = f"{prefix}{str_to_var_name(field.slug)}"
+        resolver_name = f"resolve_{field_name}"
+
+        if hasattr(schema_type, field_name):
+            logger.warning(
+                f"Unable to add the computed field {field.slug} to {schema_type._meta.slug} "
+                f"because there is already an attribute with the same name ({field_name})"
+            )
```

Comment:
Please don't use f-strings in loggers. Always use `%` interpolation passed as positional arguments, so that they are only rendered if the logger is called (lazily).
```suggestion
            logger.warning(
                "Unable to add the computed field %s to %s because there is already an attribute with the same name (%s)",
                field.slug,
                schema_type._meta.slug,
                field_name,
            )
```